app/services/users/default_categories_creator.rb

```diff
@@ -0,0 +1,17 @@
+class Users::DefaultCategoriesCreator < Struct.new(:user)
+  def call
+    default_categories_list.map { |title| create_category(title) }
+  end
+
+  private
+
+  def create_category(title)
+    user_categories.create(title: title, amount: 0, user_id: user.id)
+  end
+
+  delegate :categories, to: :user, prefix: true
```

Comment:
@marderer why did you add delegate here and not at the end of the file? Can you try that?

Reply:
@AleksSenkou here too